README.md

```diff
@@ -1,117 +1,200 @@
-merkle trees, would be advantageous when using `veractity` but is not required.
-
-## Support
+## Installation
 
-We provide pre-built native binaries for linux, mac, and windows. The
-following architectures are supported:
+Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows on the [releases](https://github.com/datatrails/veracity/releases) page.
 
-| Platform      | Architecture |
-| :--------     | -----------: |
-| MacOS(darwin) | arm64        |
-| MacOS(darwin) | x86_64       |
-| Linux         | arm64        |
-| Linux         | x86_64       |
-| Windows       | x86_64       |
-| Windows       | i386         |
+_Note_: For The Windows Subsystem for Linux (WSL), use the Linux binaries.
 
-The linux binaries can also be used in Windows Subsystem for Linux.
+| OS      | Platform  | Architecture |
+| :------ | :-------- | -----------: |
+| Mac     | `darwin`  | `arm64`      |
+| Mac     | `darwin`  | `x86_64`     |
+| Linux   | `linux`   | `arm64`      |
+| Linux   | `linux`   | `x86_64`     |
+| Windows | `windows` | `x86_64`     |
+| Windows | `windows` | `i386`       |
 
-## Installation
+1. Select the desired release from the [releases page](https://github.com/datatrails/veracity/releases).
+1. Download the archive for your host platform
+1. Extract the archive
+1. Set the file permissions
+1. Move the binary to a location on your PATH
 
+Or, follow these commands to install the latest build.
 
-Installation is a manual process:
+### Mac Install
 
-1. Download the archive for your host platform
-2. Extract the archive
-3. Set the file permissions
-4. Move the binary to a location on your PATH
+```console
+PLATFORM=$(uname -s | tr [:upper:] [:lower:])
+ARCH=$(uname -m)
+cd $TMPDIR
+curl -sLO https://github.com/datatrails/veracity/releases/latest/download/veracity_${PLATFORM}_${ARCH}.tar.gz
+tar -xf veracity_${PLATFORM}_${ARCH}.tar.gz
+chmod +x ./veracity
+mv ./veracity $HOME/.local/bin/
+veracity --help
+```
 
-For example, for the Linux or Darwin OS the following steps would be conventional
+### Linux/WSL Install
 
-```
-PLATFORM=Darwin
-ARCH=arm64
+```console
+PLATFORM=$(uname -s | tr [:upper:] [:lower:])
+ARCH=$(uname -m)
+cd /tmp
 curl -sLO https://github.com/datatrails/veracity/releases/latest/download/veracity_${PLATFORM}_${ARCH}.tar.gz
 tar -xf veracity_${PLATFORM}_${ARCH}.tar.gz
 chmod +x ./veracity
-./veracity --help
+mv ./veracity $HOME/.local/bin/
+veracity --help
 ```
 
-Set PLATFORM and ARCH according to you environment. Select the desired release
-from the [releases page](https://github.com/datatrails/veracity/releases) as VERSION (Omitting the 'v').
+#### Troubleshooting
 
-The last step should report usage information. Usual practice is to move the
-binary into a location on your $PATH. For example:
+If `veracity --help` fails, check the following:
 
-```
-mkdir -p $HOME/bin
-mv ./veracity $HOME/bin/
+confirm `` includes `.local/bin`.
```

Comment:
Should this be confirm `$PATH` rather than confirm `` ?

Reply:
Yeah, I'll fix that, thanks.

Reply:
see #22